README.md

```diff
@@ -95,6 +131,12 @@ To destroy the infrastructure, go to the `xnat-aws/provision` directory and type
 terraform destroy
 ```
 
+As before, you can also type:
+```bash
+terraform destroy -var 'create_appstream=true'
+```
+to skip the create_appstream prompt.
```

Comment:
````suggestion
As before, you can also type:

```bash
terraform destroy -var 'create_appstream=true'
```

to skip the create_appstream prompt.
````

[Fenced code blocks should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md031---fenced-code-blocks-should-be-surrounded-by-blank-lines) 🙂